tests/fingerprints/atom_pair.py

```diff
@@ -45,6 +45,23 @@ def test_atom_pair_count_fingerprint(smiles_list, mols_list):
     assert X_skfp.shape == (len(smiles_list), atom_pair_fp.fp_size)
 
 
+def test_atom_pair_count_fingerprint_normalized(smiles_list, mols_list):
+    atom_pair_fp = AtomPairFingerprint(
+        sparse=False, count=True, normalize=True, n_jobs=-1
+    )
+    X_skfp = atom_pair_fp.transform(smiles_list)
+
+    fp_gen = GetAtomPairGenerator()
+    X_rdkit = np.array([fp_gen.GetCountFingerprintAsNumPy(mol) for mol in mols_list])
+    X_rdkit_scaled = [
+        np.rint(100 * fp / mol.GetNumHeavyAtoms())
```

Comment:
Why `rint` here and `np.round` in `_scale_by_hac`?

Reply:
Previously I used `rint` in `_scale_by_hac`. It's fixed now with `round` in both.